docs/josh-commands.md

```
@@ -19,14 +19,22 @@ The four are independent and share no mutable state, so nothing is gained by run
-✔ lint (pnpm josh lint)
-✗ check (pnpm josh-app check:ci)
+✔ lint (pnpm josh lint) 9.0s
+✗ check (pnpm josh-app check:ci) 4.6s
 …
-✗ verification gate failed: lint, cspell
+✗ verification gate failed: lint, cspell (23.0s)
+```
+
+or, on a green one:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)